README.md

```diff
@@ -42,14 +42,29 @@ Then run:
 ```
 docker build -t rodekruis510/typhoonibf .
 ```
+Create a docker container
+```
+docker run --name typhoonibf rodekruis510/typhoonibf
```

Comment:
There is no point in doing this, as the container is exited immediately. The next line starts a separate new container